crawl-ref/source/tilecell.h

```diff
@@ -48,8 +93,9 @@ struct packed_cell
     tile_flavour flv;
     tileidx_t fg;
     tileidx_t bg;
+    // TODO: seems weird this isn't just part of dngn_overlay?
     tileidx_t cloud;
-    set<tileidx_t> icons;
+    packed_icons icons;
 
```

Comment:
If I'm not mistaken, dungeon overlays are things like water shorelines and ice floors from frozen ramparts which draw under monsters where as icons are things like the berserk status and draw over monsters